submissions/mxmgny/script/script.js

```diff
@@ -0,0 +1,66 @@
+const requestURL = "https://raw.githubusercontent.com/mxmgny/DOM_Practice/master/script/data.json";
+const request = new XMLHttpRequest();
```

Comment:
Try to rewrite the request for Fetch. Try to rewrite the request for Fetch. If you will need help please let me know
https://developer.mozilla.org/ru/docs/Web/API/Fetch_API/Using_Fetch

Reply:
I refactor the request code using Fetch. Please review and comment if have more question and remarks.